components/contribution-flow/PlatformTipInput.js

```diff
@@ -60,7 +66,7 @@ const getOptions = (amount, currency, intl) => {
   ];
 };
 
-const FeesOnTopInput = ({ currency, amount, quantity, fees, onChange }) => {
+const PlatformTipInput = ({ currency, amount, quantity, fees, onChange }) => {
```

Comment:
I like it!

Reply:
There's more to do on that topic, many places still use "Fees on top". I'll open a separate issue